reactor-pool/src/test/java/reactor/pool/InstrumentedPoolPoolMetricsTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2023 VMware Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.pool;
+
+import java.time.Duration;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class InstrumentedPoolPoolMetricsTest {
+
+	@Test
+	void testIsInactiveForMoreThan() {
+		InstrumentedPool.PoolMetrics metrics = new TestPool(2);
+
+		assertThat(metrics.isInactiveForMoreThan(Duration.ofMillis(1999))).isTrue();
+		assertThat(metrics.isInactiveForMoreThan(Duration.ofSeconds(2))).isTrue();
+		
+		assertThat(metrics.isInactiveForMoreThan(Duration.ofMillis(2001))).isFalse();
```

Comment:
I think testing against 1999 millis would be useful :)

Reply:
isn't it tested on row 31?

Reply:
My bad! 👓 Thanks for pointing it out :)